docs/conceptual/list.comparewith['t]-function-[fsharp].md

```diff
@@ -0,0 +1,71 @@
+---
+title: List.compareWith<'T> Function (F#)
+description: List.compareWith<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: liboz
+---
+
+# List.compareWith<'T> Function (F#)
+
+Compares two lists using the given comparison function, element by element.
+
+**Namespace/Module Path**: Microsoft.FSharp.Collections.List
+
+**Assembly**: FSharp.Core (in FSharp.Core.dll)
+
+## Syntax
+
+```fsharp
+// Signature:
+List.compareWith : ('T -> 'T -> int) -> 'T list -> 'T list -> int
+
+// Usage:
+List.compareWith comparer source1 source2
+```
+
+#### Parameters
+*comparer*
+Type: **'T -&gt; 'T -&gt;**[int](https://msdn.microsoft.com/library/025d5455-3622-4ea5-9573-3ecbd4ee1375)
+
+A function that takes an element from each of the two source lists and returns an int. If it evaluates to a non-zero value iteration is stopped and that value is returned.
+
+*source1*
+Type: **'T** [list](https://msdn.microsoft.com/library/c627b668-477b-4409-91ed-06d7f1b3e4a7)
+
+The first input list.
+
+*source2*
+Type: **'T** [list](https://msdn.microsoft.com/library/c627b668-477b-4409-91ed-06d7f1b3e4a7)
+
+The second input list.
+
+## Return Value
+Returns the first non-zero result from the comparison function. If the end of a list is reached it returns a -1 if the first list is shorter and a 1 if the second list is shorter.
```

Comment:
I understand that this is also the text [here](https://github.com/Microsoft/visualfsharp/blob/master/src/fsharp/FSharp.Core/list.fsi#L80), but this sentence doesn't really make sense and it looks like it's missing information.

It looks like it's trying to say something to the effect of:

"Returns the first non-zero result from the comparison function.  If the end of any of the two lists is reached, it will return 1 if the first list is larger, -1 if the second list is larger, or 0 if both lists are of equal length."

Could we also make a PR onto VisualFSharp to fix the comment in [list.fsi](https://github.com/Microsoft/visualfsharp/blob/master/src/fsharp/FSharp.Core/list.fsi#L79) once this is accepted?  Specifically, to break the comment about what it returns out of the `<summary>` tag and into its own `<returns>` tag?